geopandas/tests/test_geodataframe.py

```diff
@@ -51,6 +51,18 @@ def test_df_init(self):
         assert type(self.df2) is GeoDataFrame
         assert self.df2.crs == self.crs
 
+    def test_df_init_repeat_geo_col(self):
```

Comment:
Below, we have a specific `TestConstructor` class with tests specific to the constructor (the above `test_df_init` stems from before that). Can you maybe move this test to that class?